src/node/utils/cssUtils.ts

```diff
@@ -71,7 +72,7 @@ export async function compileCss(
   const postcssPlugins = postcssConfig ? postcssConfig.plugins : []
   postcssPlugins.push(require('postcss-import')())
```

Comment:
I realized we should use `unshift` instead of `push` to ensure `postcss-import` is always processed first